myst_nb/execution.py

```diff
@@ -323,7 +327,9 @@ def _converter(path):
     return result
 
 
-def nb_has_all_output(source_path: str, nb_extensions: List[str] = (".ipynb",)) -> bool:
+def nb_has_all_output(
+    source_path: str, nb_extensions: Tuple[str] = (".ipynb",)
```

Comment:
`Tuple[str]` means a tuple with only one element that is a string. For a variable number use I think `Tuple[str,...]` or maybe just `Iterable[str]` would be better

Reply:
Indeed. Changed it to `Iterable[str]` in https://github.com/executablebooks/MyST-NB/pull/328/commits/f17d12c0889b64f4ecaca79aeb694770b32660b3